roles/arteria-sequencing-report-ws/defaults/main.yml

```diff
@@ -1,10 +1,12 @@
 ---
 
 seqreport_service_repo: https://github.com/arteria-project/sequencing-report-service.git
-seqreport_service_version: v1.4.1
+seqreport_service_version: v1.5.0-rc2
```

Comment:
There's a new release candidate in town! :cowboy_hat_face: https://github.com/arteria-project/sequencing-report-service/releases/tag/v1.5.0-rc3

Reply:
```suggestion
seqreport_service_version: v1.5.0-rc3
```